### ISSUE TITLE
Manual Test the Polices & Confirm Within PolicyBuilderModule.js

### ISSUE THREAD

**Issue by mle6366** [ExpanseLLC/lambda_authorizer#17]
Here is what is actually being returned right now (acct id omitted). I see some problems with these as well, especially the DENY one.

```
{
    "policyDocument": {
        "Effect": "DENY",
        "Version": "2012-10-17",
        "Action": "execute-api:Invoke",
        "Resource": [
            "arn:aws:execute-api:false:false:kvmxspwm7g/*/GET/"
        ]
    },
    "principalId": ""
}
```

``````
{
  "policyDocument": {
    "Effect": "ALLOW",
    "Version": "2012-10-17",
    "Action": "execute-api:Invoke",
    "Resource": [
      "arn:aws:execute-api:us-west-2:xxxxx:kvmxspwm7g/*/GET/"
    ]
  },
  "principalId": "erickson.meghan@gmail.com"
}```
``````

**Comment by mle6366** [comment]
Policy Simulator Link: 
https://policysim.aws.amazon.com/home/index.jsp?#

**Comment by mle6366** [comment]
Yeah . . . missing the Statement. Whoops.

**Comment by ryanwscott** [comment]
Resolved with #18
